app/policies/application_policy.rb

```diff
@@ -6,6 +6,71 @@ def initialize(user, record)
     @record = record
   end
 
+  def current_person
+    @current_person ||= user.person
+  end
+
+  # TODO: Remove the following line after the complete implementation of the methods
+  # MARCO's reference commit - https://github.com/ideacrew/enroll/commit/d70f1722dc9c8cc27d3ccdc9a5b1e77e2d34c7d8
+
+  # START - Individual Market related methods
+  def individual_market_primary_family_member?(family)
+    family.primary_consumer && current_person == family.primary_person
```

Comment:
What is family.primary_consumer and how is it different from family.primary_family_member?  

If this syntax stands, add parenthesis after the && to more clearly indicate the logi